src/ompl/base/src/Planner.cpp

```diff
@@ -134,6 +134,11 @@ ompl::base::PlannerStatus ompl::base::Planner::solve(double solveTime)
         return solve(timedPlannerTerminationCondition(solveTime, std::min(solveTime / 100.0, 0.1)));
 }
 
+ompl::base::PlannerStatus ompl::base::Planner::solve(time::duration solveTime)
+{
+    return solve(timedPlannerTerminationCondition(solveTime));
+}
```

Comment:
I don't understand why `solve(double)` and `solve(time::duration)` are both needed and do slightly different things. If they can really have the same behavior, then you can pass in `solve(time::seconds(solveTime.seconds))` to the existing method.